### PR TITLE
Add changelog entry for firestore::LoadBundle()

### PR DESCRIPTION
**Pull request by wu-hui**

### CODE REVIEW

File: release_build_files/readme.md [hunk 1]
```diff
@@ -587,6 +587,9 @@ code.
 	with version 8.x of the Firebase iOS SDK.
     -   General (iOS): A Database URL is no longer required to be present in
         GoogleService-Info.plist when not using the Real Time Database.
+    -   Firestore: Added `Firestore::LoadBundle` to enable loading Firestore Data
+        Bundles to the SDK cache. `Firestore::NamedQuery` will be availabe in next
```
Comment (var-const) [inline release_build_files/readme.md:591]: Nit: `s/availabe/available/`.

Comment (var-const) [inline release_build_files/readme.md:591]: Nit: add `the` (`in the next release`).

Comment (var-const) [inline release_build_files/readme.md:591]: Hmm, I think it should be `s/to/into/` (i.e., `loading... into the cache`). @jonsimantov Jon, which one looks better to you?

**Review by var-const — CHANGES_REQUESTED** [review]
Please unrevert the removal of `NamedQuery`.

File: release_build_files/readme.md (hunk 1 above)
Comment (var-const) [reply release_build_files/readme.md:591]: On a second thought, how about `a next release`?

Comment (wu-hui) [reply release_build_files/readme.md]: Done.

Comment (wu-hui) [reply release_build_files/readme.md]: Done.

Comment (wu-hui) [reply release_build_files/readme.md]: `into` is better, IMO.

**Review by var-const — APPROVED** [review]

**Comment by var-const** [comment]
@wu-hui Approved, but please check with @jonsimantov Jon that these notes should be under the `8.0.0` release.

**Review by jonsimantov — APPROVED** [review]

**Pull request merged**